### ISSUE TITLE
Allow a way to use remark plugins

### ISSUE THREAD

**Issue by hunterliu1003** [nuxt/content#39]
### Is your feature request related to a problem? Please describe.
I would like to use [remark](https://github.com/remarkjs/remark) plugins in markdown content, for example:

For instance, [@nuxtjs/markdownit](https://www.npmjs.com/package/@nuxtjs/markdownit) can config in `nuxt.config.js`:
```js
markdownit: {
  use: [
    'markdown-it-div',
    'markdown-it-attrs'
  ]
}
```
and [Vuepress](https://vuepress.vuejs.org/config/#markdown-plugins) can config in `.vuepress/config.js`:

```js
markdown: {
  plugins: {
    'markdown-it-bar': {
      // provide options here
    }
  }
}
```

### Describe the solution you'd like
In short: a way to use remark plugins

```js
export default {
  content: {
    use: [
      [
        'remark-attr', { 
          // provide options here 
        }
      ],
      // else plugins
    ]
  }
}
```

**Comment by benjamincanac** [comment]
Hey @hunterliu1003,

You can now use plugins in [v1.2.0](https://github.com/nuxt/content/releases/tag/v1.2.0).

Here is the link to the documentation: https://content.nuxtjs.org/configuration#markdown

**Comment by adamdehaven** [comment]
@benjamincanac can you show an example of how to actually use a remark plugin (e.g. `remark-directive`)? I've read through the documentation several times but can't seem to get a remark plugin up and working properly. The documentation could really use a working example.
